realm/realm-jni/build.gradle

```diff
@@ -248,6 +248,67 @@ task deployCore(group: 'build setup', description: 'Deploy the latest version of
     }
 }
 
+task assembleRobolectric(description: "Build the Robolectric shared library for Darwin") {
+    def staticLibPath = "/usr/local/lib/librealm.a"
```

Comment:
I see. How about adding the environment variable `REALM_CORE`? If it is not set correctly, we use `/usr/local/lib/` as the default path.